workshop/exercise-2/README.md

```diff
@@ -14,7 +14,7 @@ export PATH=$PWD/istio-<version-number>/bin:$PATH
 
 5. Install Istio on the Kubernetes cluster. Istio is deployed in the Kubernetes namespace `istio-system`. Since in a later exercise we will try out the mutual TLS features, we install the `istio-demo-auth.yaml` here.
 ```bash
-kubectl apply -f install/kubernetes/istio-demo.yaml
+kubectl apply -f install/kubernetes/istio-demo-auth.yaml
```

Comment:
we should not teach people to install demo-auth here.

Reply:
should we have them install just `istio-demo.yaml` instead?

note that this may cause issues with [Exercise 7](https://github.com/IBM/istio101/blob/master/workshop/exercise-7/README.md)

Reply:
yes istio-demo.yaml +1.  This should not cause any issue in exercise 7, because we should be use the new auth policy to enable mTLS